module-simple/src/main/java/domainapp/modules/simple/dom/so/SimpleObject.java

```diff
@@ -45,49 +57,164 @@ public static SimpleObject withName(String name) {
         return simpleObject;
     }
 
-    public static class ActionDomainEvent extends SimpleModule.ActionDomainEvent<SimpleObject> {}
-
+	/*
+	 * These dependencies are automatically injected 
+	 * into an entity when persisting it or retrieving 
+	 * it from the persistence layer.
+	 */
     @Inject RepositoryService repositoryService;
     @Inject TitleService titleService;
-    @Inject MessageService messageService;
-
+    @Inject MessageService messageService;    
+    
     private SimpleObject() {
     }
 
+    /*
+     * This method is part of the programming model of Apache Isis.
```

Comment:
I would have the first sentence describe what the method does.
Then <p> and start a new paragraph, where you can describe the programming model stuff.  And a <p> for each subsequent paragraph.